File: src/funciones/lambda/lambda3.py
from typing import List, TypeVar
S = TypeVar(str)

def recorrido(cadena: str) -> List[S]:
    array = []
    for i in cadena:
        array.append(i)
    return array
print(recorrido(cadena = str(input('Di una Cadena: '))))


#lambda
array = []
recorrido2 = (lambda cadena: [print(array.append(i)) for i in cadena])
print(recorrido2('HOLA QUE TAL'))
 

File: src/funciones/lambda/lambda3.js
function recorrido(cadena) {
    array = [];
    for (let i = 0; i <= cadena.length; i++) {
        array.push(cadena[i]);
    }
    return array;
}
console.log(recorrido('HOLA QUE TAL'));

recorrido1 = (cadena => {
    let array = [];
    for (let i = 0; i <= cadena.length; i++) {
        array.push(cadena[i]);

    }
    return array;
});
console.log(recorrido1('HOLA QUE TAL'));
